oarepo_runtime/services/schema/validation.py

```diff
@@ -25,6 +20,15 @@ def validate_identifier(value):
             # the normalize_pid library has problems with isbn - does not raise an exception
             # but returns an empty string
             raise ValueError()
+        
```

Comment:
We should just validate the isbn here, not change it (especially not from 10 to 13 base, though they are equivalent - it should match the value that is printed on the publication)